stable/opsportal/templates/landing.yaml

```diff
@@ -52,7 +52,7 @@ spec:
     spec:
       containers:
         - name: opsportal-landing
-          image: nginx:1.17-alpine
+          image: nginx:1.18-alpine
```

Comment:
```suggestion
          image: nginx:1.20-alpine
```
or even
```suggestion
          image: nginx:1.20.0-alpine
```
or even
```suggestion
          image: nginx@sha256:e015192ec74937149dce3aa1feb8af016b7cce3a2896246b623cfd55c14939a6
```

Reply:
I agree that we should bump further to catch up with upstream but doing a small bump lowers the risks of the change, which aligns better with where we are in the release process.